clients/js/src/utils.ts

```diff
@@ -37,7 +37,8 @@ export function repack(value: unknown): any {
 }
 
 export async function handleError(error: unknown) {
-  if (error instanceof Response) {
+  // @ts-ignore
```

Comment:
We no longer use this function so you may need to update this PR to accommodate the new changes.